src/Endpoint/EndpointBase.php

```diff
@@ -6,6 +6,7 @@
 
 use OpenEuropa\CdtClient\Contract\RestInterface;
 use OpenEuropa\CdtClient\Traits\ConfigurationAwareTrait;
+use OpenEuropa\CdtClient\Traits\SerializerAwareTrait;
```

Comment:
Love the idea with the trait, but we need to clean up the use statements

Reply:
Thanks for pointing this out, I try to check this before every commit, but sometimes I miss something. It is not in the pipeline because there are different standards for defining fully qualified namespaces in `@var`, `@param` and `@return`, and the IDE reports a lot of problems.